src/main/scala/temple/generate/language/service/go/GoServiceGenerator.scala

```diff
@@ -0,0 +1,56 @@
+package temple.generate.language.service.go
+
+import temple.generate.language.service.ServiceGenerator
+import temple.generate.language.service.adt._
+import temple.utils.FileUtils.{File, FileContent}
+import temple.utils.StringUtils.tabIndent
+
+object GoServiceGenerator extends ServiceGenerator {
+
+  private def generatePackage(packageName: String): String = s"package $packageName\n\n"
+
+  private def generateImports(serviceName: String, module: String, comms: Boolean): String = {
```

Comment:
To the uninitiated, it might be clear what `comms` means. Might be worth adding a comment